build_wasm_wrappers_docker.sh

```diff
@@ -1,3 +1,8 @@
 #!/bin/bash
 echo "Building WebAssembly wrappers inside Docker container"
-docker run -it --rm -u $(id -u):$(id -g) -v `pwd`:/src emscripten/emsdk:2.0.14 /src/wasm_src/build_wrappers.sh
+if [ -t 0 ] && [ -t 1 ]; then
+  docker run -it --rm -u $(id -u):$(id -g) -v `pwd`:/src emscripten/emsdk:2.0.14 /src/wasm_src/build_wrappers.sh
+else
+  # Non interactive version needed by Jenkins as no TTY is available 
+  docker run --rm -u $(id -u):$(id -g) -v `pwd`:/src emscripten/emsdk:2.0.14 /src/wasm_src/build_wrappers.sh
```

Comment:
is there any reason why we should continue to use `-it` under normal circumstances?

Reply:
wait, nevermind. If we don't have `-it` we don't get output, right?